### ISSUE TITLE
Commands to access cluster do not work

### ISSUE THREAD

**Issue by nikhil550** [IBM-Bluemix-Docs/containers#2434]
The commands to access a cluster of IKS using the command seem to be out of date.

On the front page of your cluster - https://cloud.ibm.com/kubernetes/clusters/

The first command is: `curl -sL https://ibm.biz/idt-installer | bash`

However, this command downloads and outdated CLI that does not allow you to login to your instance on IBM Cloud. The command needs to be updated for the next command
`ibmcloud login -a cloud.ibm.com -r us-south -g default` to work.

The command to get your cluster information `ibmcloud ks cluster-config --cluster niks-cluster` returns `'ks' is not a registered command. See 'ibmcloud help'.`

Instead, the command `ibmcloud cs cluster-config --cluster niks-cluster` is what is needed to retrieve the necessary config.

**Comment by Nadine2016** [comment]
@nikhil550 so the command is correct from what I can tell. Can you share the installation log? If you don't have it anymore, could you share your output of `ibmcloud plugin list`?

**Comment by nikhil550** [comment]
``` ibmcloud plugin list
Listing installed plug-ins...

Plugin Name                        Version   Status   
sdk-gen                            0.1.13       
cloud-functions/wsk/functions/fn   1.0.32       
cloud-object-storage               1.0.0        
container-registry                 0.1.394      
container-service                  0.1.528   Update Available   
dev                                2.3.0        
icp                                2.1.343      
logging-cli                        1.0.5     Update Available   

```

**Comment by Nadine2016** [comment]
@nikhil550 can you please remove the `container-service` plug-in (`ibmcloud plugin uninstall container-service`) and then rerun `curl -sL https://ibm.biz/idt-installer | bash` I am wondering if there is some kind of issue with the old plug-in.

**Comment by nikhil550** [comment]
Hi @Nadine2016, I ran the command `ibmcloud plugin uninstall container-service`, and then the instructions ran as currently documented. Thank you. I will leave this issue open and let you investigate.

**Comment by Nadine2016** [comment]
@nikhil550 so we confirmed that because you were using a very old plug-in, the installer thought that you had it installed already and that's why the installer wanted to update the plug-in. but because the plug-in was so old, no update was available for `container-service`. and that's why the plug-in stayed in this old state. 

we deprecated the plug-in about 1 year ago, which is why we don't want to pursue fixing that. I am also not aware of any other user that ran into this issue, which obviously does not mean that this never happened before. But it seems to not be a very common issue, specifically now that the majority of users already have the new plug-in installed.
